src/components/BackToTop.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div :class="{ show: visible }" class="scroll-to-top" @click="backToTop">
+    <img src="@/assets/up-arrow.png" alt="arrow" width="50px" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ScrollToTop",
+  data() {
+    return {
+      visible: false,
+    };
+  },
+  methods: {
+    trackScroll() {
```

Comment:
I propose we use https://vueuse.org/core/usescroll/ so that code that tracks scrolling is easier to read.

@Zeebro2223 What do you think?